src/library_dylink.js

```diff
@@ -240,12 +240,18 @@ var LibraryDylink = {
   _emscripten_dlopen_js__deps: [function() { error(dlopenMissingError); }],
   _dlsym_js__deps: [function() { error(dlopenMissingError); }],
   _dlsym_catchup_js__deps: [function() { error(dlopenMissingError); }],
+  dlopen__deps: [function() { error(dlopenMissingError); }],
+  _emscripten_dlopen__deps: [function() { error(dlopenMissingError); }],
+  __dlsym__deps: [function() { error(dlopenMissingError); }],
```

Comment:
What does a function expression in `__deps` mean/do?

Reply:
This magic basically means that users see errors at build time when they use those symbols.